rasterio/crs.pyx

```diff
@@ -98,13 +98,15 @@ cdef class CRS:
         self._data = {}
         self._epsg = None
         self._wkt = None
+        self._geodetic_crs = None
 
         if initialdata or kwargs:
             tmp = CRS.from_dict(initialdata=initialdata, **kwargs)
             self._osr = OSRClone(tmp._osr)
             self._wkt = tmp._wkt
             self._data = tmp.data
             self._epsg = tmp._epsg
+            self._geodetic_crs = tmp._geodetic_crs
```

Comment:
```suggestion
```

This will always be None in practice, so we can omit this if we want. Might be for the best so that we don't signal that we're trying to propagate a geodetic CRS.